static/index.html

```diff
@@ -1,9 +1,9 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <title>FoxLink</title>
+    <title>FoxBox</title>
```

Comment:
Since the repo is called foxbox, and neither of the two names have been decided on anyway, I changed this to match the repo name for consistency.